.github/workflows/swift-toolchain.yml

```diff
@@ -303,6 +311,155 @@ jobs:
           name: sqlite-${{ matrix.os }}-${{ matrix.arch }}-${{ inputs.swift_toolchain_sqlite_version }}
           path: ${{ github.workspace }}/BuildRoot/Library/sqlite-${{ inputs.swift_toolchain_sqlite_version }}/usr
 
+  ds2_tools:
+    runs-on: ${{ inputs.default_build_runner }}
```

Comment:
```suggestion
    # TODO: Build this on macOS or make an equivalent Mac-only job
    if: inputs.build_os == 'Windows'
    runs-on: ${{ inputs.default_build_runner }}
```

Reply:
Do we have an open issue tracking macos build that I can tag in the TODO?

Reply:
I do but it's in the downstream repo so the tags don't work. Just set it the same way I've been setting it up elsewhere in the file, this will eventually go away anyway.